sklearn/linear_model/_ridge.py

```diff
@@ -2323,6 +2327,28 @@ def fit(self, X, y, sample_weight=None, **params):
         _raise_for_params(params, self, "fit")
         cv = self.cv
 
+        # TODO(1.7): Remove in 1.7
+        # Also change `store_cv_results` default back to False
+        if self.store_cv_values != "deprecated":
+            if self.store_cv_results is not None:
+                raise ValueError(
+                    "Both 'store_cv_values' and 'store_cv_results' were set. "
+                    "'store_cv_values' was deprecated in version 1.5 and will be "
+                    "removed in 1.7. To avoid this error, only set 'store_cv_results'."
+                )
+            warnings.warn(
+                (
+                    "'store_cv_values' was renamed to 'store_cv_results' in version "
+                    " 1.5 and will be removed in 1.7."
+                ),
+                FutureWarning,
+            )
+            self._store_cv_results = self.store_cv_values
+        elif self.store_cv_results is None:
+            self._store_cv_results = False
```

Comment:
why is it not False by default?

Reply:
Its just to know if `store_cv_results` has been explicitly set so we can give the appropriate error/warning. 
Its the same structure to https://github.com/scikit-learn/scikit-learn/pull/27718 and https://github.com/scikit-learn/scikit-learn/pull/28471